api.py

```diff
@@ -1,21 +1,24 @@
+import pickle
 from fastapi import FastAPI
 from pydantic import BaseModel
-import pickle
 import pandas as pd
 
 app = FastAPI()
 
-class ScoringItem(BaseModel): 
-    YearsAtCompany: float 
-    EmployeeSatisfaction: float 
-    Position:str
-    Salary: int
 
-with open('model/model.pkl', 'rb') as f: 
+class ScoringItem(BaseModel):
+    sepal_length: float
+    sepal_width: float
+    petal_length: float
+    petal_width: float
+
+
+with open("model/model.pkl", "rb") as f:
     model = pickle.load(f)
 
-@app.post('/')
-async def scoring_endpoint(item:ScoringItem): 
+
+@app.post("/")
+async def scoring_endpoint(item: ScoringItem):
```

Comment:
the method name in a controller should be very similar than the API definition
```suggestion
async def scoring_endpoint(item: ScoringItem):
```